core/templates/services/context.service.ts

```diff
@@ -218,9 +218,7 @@ export class ContextService {
   // Returns a string representing the explorationId (obtained from the
   // URL).
   getExplorationId(): string {
-    if (this.explorationId) {
-      return this.explorationId;
-    } else if (!this.isInQuestionPlayerMode()) {
```

Comment:
Why has this been removed?

Reply:
In general, when making PRs related to increasing test coverage, please avoid making changes to current code that affects functionality unless absolutely required.

Reply:
Reverted back changes. Done!